transport/http/constants.go

```diff
@@ -155,3 +155,9 @@ var (
 		_http2SchemePseudoHeader,
 	}
 )
+
+// Tracing header keys that must be preserved in HTTP headers.
+const (
+	UberTraceIDHeader = "uber-trace-id"
+	UberCtxHeader     = "uberctx-"
```

Comment:
TBH I don't really like that uber-specific stuff is here, but we're using deprecated open source library that has all these constants, so we basically partially merging that library inside a yarpc-go. And eventually we'll get rid of outdated dependency.

(It's not an call-to-action comment, just leaving it here for the history.)